doc/installation.rst

```diff
@@ -33,17 +33,17 @@ such as |PyInstaller| in each of them.
 (For more on the uses of virtualenv, see :ref:`Supporting Multiple Platforms` below.)
 
 When pip-Win is working, enter this command in its Command field
-and click Run:
+and click Run::
 
-  ``venv -c -i  pyi-env-name``
+    venv -c -i pyi-env-name
 
 This creates a new virtual environment rooted at ``C:\Python\pyi-env-name``
 and makes it the current environment.
 A new command shell
 window opens in which you can run commands within this environment.
-Enter the command
+Enter the command::
```

Comment:
[Quick reStructuredText](http://docutils.sourceforge.net/docs/user/rst/quickref.html#literal-blocks) says: "The :: will be omitted if it is preceded by whitespace." So I suggest adding a space in front, since the sentence continues over the literal block. Same for the other places.